Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -376,6 +419,18 @@ void MediaPlayerPrivateGStreamerMSE::getSupportedTypes(HashSet<String, ASCIICase
 
 MediaPlayer::SupportsType MediaPlayerPrivateGStreamerMSE::supportsType(const MediaEngineSupportParameters& parameters)
 {
+    static unsigned mediaMaxWidth = 0;
+    static unsigned mediaMaxHeight = 0;
+    static unsigned mediaMaxFrameRate = 0;
+#ifdef VIDEO_DECODING_LIMIT
+    static bool isVideoDecoderLimitDefaultParsed = false;
+    if (!isVideoDecoderLimitDefaultParsed) {
+        isVideoDecoderLimitDefaultParsed = true;
+        if (!videoDecoderLimitsDefaults(mediaMaxWidth, mediaMaxHeight, mediaMaxFrameRate))
+            GST_ERROR("Parsing VIDEO_DECODING_LIMIT failed");
```

Comment:
Once you set defaults, you can just `GST_WARNING` here because things are going to "work" but maybe not as expected. A warning should be enough IMHO but also, please call `ASSERT_NOT_REACHED()` in that case so we have a partially working thing in release but a crash in debug.

Reply:
If I use ASSERT_NOT_REACHED() I have style check error:

`ERROR: Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp:429:  ASSERT_NOT_REACHED() statement fallthrough may result in unexpected code execution.  [security/assertion_fallthrough] [4]`

Any idea how to fix it?

Reply:
This is because `ASSERT_NOT_REACHED()` expands to nothing in release builds. Try `RELEASE_ASSERT_NOT_REACHED()` instead, which will always work regardless of the build type.

Reply:
I think that(expand to nothing in release) is what we want. @calvaris : do you think I should use RELEASE_ASSERT_NOT_REACHED() here?

Reply:
No, definitely do not use `RELEASE_ASSERT_NOT_REACHED()`, that will crash in release mode. We just want to crash in debug mode.